docs/plugins/first_plugin.md

```diff
@@ -21,6 +21,22 @@ recommended**). If you are new to virtual environments, we recommend
 [installing miniconda][miniconda] and [creating a new environment with
 python][python_env].
 
+Even though your plugin does not need to directly depend on the `napari`
+package, and
+[should not depend on a specific Qt backend](best-practices-no-qt-backend),
+you will need a full installation of napari in your active Python
+environment to use and test your plugin.
+If napari is not already installed and working, consider installing it
+with `pip`
+
+```sh
+pip install "napari[all]>=0.4.13"
+```
+
+where `[all]` means that the default Qt backend will be installed and
+`0.4.13` is the minimum version of napari that supports `npe2`.
```

Comment:
```suggestion
`0.4.13` is the minimum version of napari that supports `npe2` If you have never installed napari before, you may wish to look at our more detailed [installation guide](link-to-guide).
```

I'm wary of providing such succinct instructions for installation to new users so I'd like to link out to the full guide here.

Reply:
sounds good.  maybe let's just do that for the whole thing?  That is:

> Even though your plugin does not need to directly depend on the napari package, and [should not depend on a specific Qt backend](https://github.com/napari/napari/blob/e4cb8ccd60d6973a585191e053346dc5aad25bad/docs/plugins/best-practices-no-qt-backend), you will need a full installation of napari in your active Python environment to use and test your plugin.  See the [installation guide]() for details.

?

Reply:
Yup, I think that works!

Reply:
Thanks for the feedback @DragaDoncila. I used the wording from @tlambert03, so I think this is resolved now.